arcgis-ios-sdk-samples/Layers/Show labels on layers/ShowLabelsOnLayersViewController.swift

```diff
@@ -0,0 +1,99 @@
+        let featureTableURL = URL(string: "https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/USA_Congressional_Districts_analysis/FeatureServer/0")!
+        
+        // create a feature table from the URL
+        let featureTable = AGSServiceFeatureTable(url: featureTableURL)
+        
+        // set the map viewpoint to show the layer
+        mapView.setViewpointCenter(AGSPoint(x: -10840000, y: 4680000, spatialReference: .webMercator()), scale: 20000000)
+        
+        // create a feature layer from the table
+        let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+        // add the layer to the map
+        map.operationalLayers.add(featureLayer)
+        
+        // create label definitions for the two groups
+        let demDefinition = makeLabelDefinition(party: "Democrat", color: .blue)!
+        let repDefinition = makeLabelDefinition(party: "Republican", color: .red)!
+        
+        // add the label definitions to the layer
+        featureLayer.labelDefinitions.addObjects(from: [demDefinition, repDefinition])
+        // turn on labelling
+        featureLayer.labelsEnabled = true
+    }
+    
+    /// Creates a label definition for the given PARTY field value and color.
+    private func makeLabelDefinition(party: String, color: UIColor) -> AGSLabelDefinition? {
+        
+        // The JSON syntax reference for AGSLabelDefinition.fromJSON(_:) can be found here:
+        // https://developers.arcgis.com/web-map-specification/objects/labelingInfo/
+        
+        /// The styling for the label.
+        let textSymbol = AGSTextSymbol()
+        textSymbol.size = 12
+        textSymbol.haloColor = .white
+        textSymbol.haloWidth = 2
+        textSymbol.color = color
+        // the object must be converted to a JSON object
+        let textSymbolJSON = try! textSymbol.toJSON()
+        
+        /// A SQL WHERE statement for filtering the features this label applies to.
+        let whereStatement = "PARTY = '\(party)'"
+        
+        /// An expression that specifies the content of the label using the table's attributes.
+        let expression = "$feature.NAME + ' (' + left($feature.PARTY,1) + ')\\nDistrict' + $feature.CDFIPS"
+        
+        /// The root JSON object defining the label.
+        let labelJSONObject: [String: Any] = [
+            // see https://developers.arcgis.com/web-map-specification/objects/labelExpressionInfo/
+            "labelExpressionInfo": [
+                "expression": expression
+            ],
+            "labelPlacement": "esriServerPolygonPlacementAlwaysHorizontal",
+            "where": whereStatement,
+            "symbol": textSymbolJSON
+        ]
+        
+        do {
+            // create and return a label definition from the JSON object
+            return try AGSLabelDefinition.fromJSON(labelJSONObject) as? AGSLabelDefinition
+        }
+        catch {
+            presentAlert(error: error)
+            return nil
```

Comment:
Returning `nil` here will cause a crash where the function is called and the returned value is force unwrapped.  Here again I think avoiding a crash would be good.